Applications/Spire/Source/Ui/SecurityBox.cpp

```diff
@@ -33,8 +33,13 @@ struct SecurityBox::SecurityQueryModel : ComboBox::QueryModel {
           }
         }();
         auto result = std::vector<std::any>();
+        auto securities = std::unordered_set<Security>();
         for(auto& value : matches) {
-          result.push_back(std::any_cast<SecurityInfo&>(value).m_security);
+          auto& security = std::any_cast<SecurityInfo&>(value).m_security;
+          if(!securities.contains(security)) {
```

Comment:
This does two lookups when you can perform this with a single lookup:

https://en.cppreference.com/w/cpp/container/unordered_set/insert

    if(securities.insert(security).second) {
      result.push_back(security);
    }